app-drawer/app-drawer.html

```diff
@@ -137,7 +139,8 @@
       transition: opacity 0.2s ease;
 
       -webkit-transform: translateZ(0);
-      transform:  translateZ(0);
+      transform: translateZ(0);
+      will-change: opacity;
```

Comment:
Probably still needed to force a layer on IE/Edge though http://caniuse.com/#feat=will-change